css/style.css

```diff
@@ -0,0 +1,77 @@
+@import "https://fonts.googleapis.com/css?family=Crete Round|Inter";
+
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: 'crete round', sans-serif;
+}
+
+body {
+  width: 100%;
+  height: auto;
+  position: relative;
+}
+
+h1 {
+  color: #ff6b00;
+  font-size: 56px;
+  align-self: center;
+  justify-content: center;
+  margin: 6px 24px;
+  font-family: 'crete round', sans-serif;
+}
+
+.toolbar {
+  height: 50px;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  color: white;
+  align-items: center;
```

Comment:
- [ ] The margin-top on your logo and hamburger needs a little adjustment to match that of the figma design. Making this change will give your users a better experience with your web app.

## Your view:
![image](https://user-images.githubusercontent.com/50990925/219270387-8559f745-b20c-4839-8f6e-87965b03b104.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/219270554-df60e131-bf69-4dbb-88df-d43c704ba25f.png)